src/components/elements/BeholdningPanel/BeholdningPanel.test.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 import { render } from '@testing-library/react';
 import {BeholdningPanel} from './BeholdningPanel';
+import {formatAmount} from "../../../common/utils";
 
 it('renders the Beholdning panel with correct heading and correct formatted assets', () => {
     const panel = render(<BeholdningPanel data={{'beholdning': "1200000"}}/>);
 
     expect(panel.getByRole("heading")).toHaveTextContent("beholdning-din-pensjonsbeholdning-i-folketrygden");
-    expect(panel.getByTestId("assets")).toHaveTextContent("1 200 000");
+    expect(panel.getByTestId("assets").textContent).toEqual("kr " + formatAmount(1200000));
```

Comment:
Ser ut som man bruker funksjonen som skal testes (`formatAmount`)?

Reply:
Av en eller annen grunn blir beløpet formatert som engelsk (med komma-separator) på Github, mens det blir formatert korrekt lokalt.   Derfor la jeg til formateringsfunksjonen, for at det skal bli likt overalt

Reply:
Men, ja det blir kanskje litt rart - når testen skulle sjekke beløp og formatet.

Reply:
Hmm, en mulighet er kanskje å ha en separat `formatAmount`-funksjon for test (duplisert kode altså). Da vil testen mest sannsynlig feile om man ved et uhell endrer den opprinnelige funksjonen. Du får vurdere det.
Men nå får man ihvertfall testet headingen og at det står "kr" foran.

Reply:
Det er github actions som ikke støtter localen `nb-NO`.
Den støtter derimot `no-NO`.
https://github.com/navikt/pensjon-selvbetjening-opptjening-frontend/pull/50/checks?check_run_id=3764303665